hack/tools/release/notes.go

```diff
@@ -403,8 +446,82 @@ REPLACE ME: A couple sentences describing the deprecation, including links to do
 
 	fmt.Println("")
 	fmt.Println("_Thanks to all our contributors!_ 😊")
+}
 
-	return 0
+func parseWeeklyUpdate(commits []*commit, merges map[string][]string, commitRange string, lastDay string) {
+	for _, c := range commits {
+		body := trimTitle(c.body)
+		var key, prNumber, fork string
+		switch {
+		case strings.HasPrefix(body, ":sparkles:"), strings.HasPrefix(body, "✨"):
+			key = features
+			body = strings.TrimPrefix(body, ":sparkles:")
+			body = strings.TrimPrefix(body, "✨")
+		case strings.HasPrefix(body, ":bug:"), strings.HasPrefix(body, "🐛"):
+			key = bugs
+			body = strings.TrimPrefix(body, ":bug:")
+			body = strings.TrimPrefix(body, "🐛")
+		case strings.HasPrefix(body, ":book:"), strings.HasPrefix(body, "📖"):
+			key = documentation
+			body = strings.TrimPrefix(body, ":book:")
+			body = strings.TrimPrefix(body, "📖")
+			if strings.Contains(body, "CAEP") || strings.Contains(body, "proposal") {
+				key = proposals
+			}
+		case strings.HasPrefix(body, ":seedling:"), strings.HasPrefix(body, "🌱"):
+			key = other
+			body = strings.TrimPrefix(body, ":seedling:")
+			body = strings.TrimPrefix(body, "🌱")
+		case strings.HasPrefix(body, ":warning:"), strings.HasPrefix(body, "⚠️"):
+			key = warning
+			body = strings.TrimPrefix(body, ":warning:")
+			body = strings.TrimPrefix(body, "⚠️")
+		default:
+			key = unknown
+		}
+
+		body = strings.TrimSpace(body)
+		if body == "" {
+			continue
+		}
+		body = fmt.Sprintf("\t - %s", body)
+		_, _ = fmt.Sscanf(c.merge, "Merge pull request %s from %s", &prNumber, &fork)
+		if key == documentation {
+			merges[key] = append(merges[key], prNumber)
+			continue
+		}
+		merges[key] = append(merges[key], formatMerge(body, prNumber))
+	}
+
+	// TODO Turn this into a link (requires knowing the project name + organization)
```

Comment:
Could this be solved by just taking the URL - or project name and org - as args to the command? Maybe better to do that up front.

Reply:
Did you add the TODO? I'm not sure what it's asking to turn into a link, but it seems closer to L557